src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesSchedulerIT.java

```diff
@@ -988,6 +988,34 @@ public void testConcurencyPolicy(String concurrencyPolicy) {
 		kubernetesScheduler.unschedule(cronJob.getMetadata().getName());
 	}
 
+	@ParameterizedTest
+	@ValueSource(strings = {"100", "86400"})
```

Comment:
Nice test. I can think of a couple of other cases we could add to improve coverage:
* invalid ttl value (deployer prop)
* invalid ttl value (server default prop)
* deployer prop set and server default prop set (deployer prop wins)
* no deployer prop set but server default prop set
* no deployer prop set and no server default prop set
Wdyt?

Reply:
Well done.